tests/view_tests/test_image_routes.py

```diff
@@ -29,10 +28,9 @@ def test_post_image_route_with_file_bad_extension(client):
     assert response.status_code == 400
     assert b'Error with request: File extension not allowed.' in response.data
 
-@patch('werkzeug.datastructures.FileStorage.save')
-def test_post_image_route_with_file(mock_save, client):
+def test_post_image_route_with_file(monkeypatch, client):
+    monkeypatch.setattr('werkzeug.datastructures.FileStorage.save', lambda a, b: "" )
```

Comment:
```suggestion
    monkeypatch.setattr('werkzeug.datastructures.FileStorage.save', lambda: "" )
```

Your lambda doesn't need any arguments.

Reply:
It did